scripts/generate_style.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+import * as Style from "../src/js/style.js";
+
+let style = Style.build(
+  "https://6ug7hetxl9.execute-api.us-east-2.amazonaws.com/data/v3.json",
+  "https://zelonewolf.github.io/openstreetmap-americana/sprites/sprite",
```

Comment:
If we’re comfortable with hard-coding the sprite base URL here, we might as well do it for the Web code path too, rather than dynamically building the URL based on the containing webpage’s URL.

Reply:
I haven't fully thought through all the parts of what a "deployable" package would look like. The americana.js file is really becoming the demo_map.js.  The fact that the sprites URL gets baked into the style.json is kind of a problem for deployability. At least it would be one less thing to have to deal with when the repo changes. I guess ultimately we would have to provide some kind of script for people to run to generate the necessary artifacts.

Reply:
#70 has multiple possible answers. Americana’s “demo map” is already deployable as an iframe embedded in a webpage. Exporting `Style.build()` makes this package deployable to a project that uses NPM to build the webpage.

But it looks like you’re working towards an even tighter level of integration, a style JSON that can be dropped into a webpage that already has GL JS installed but doesn’t use NPM. There will be an expectation that the style is _not_ self-contained but rather fully hosted except for the style JSON. In that context, there would probably be little demand for the ability to configure these URLs.